Sito/contatti.html

```diff
@@ -68,7 +68,7 @@ <h2>Orari</h2>
           <h2>Indirizzo</h2>
       </div>
       <p>35131 Padova (PD) - Piazza Eremitani, 404/2  </p>
-      <p>Clicca sulla mappa per aprirla su Google Maps </p>
+      <p class="h_info" >Clicca sulla mappa per aprirla su Google Maps </p>
 
       <div id="info_stradali">
           <a href="https://www.google.it/maps/place/Piazza+Eremitani,+35121+Padova+PD/@45.4105663,11.8767516,17z/data=!3m1!4b1!4m5!3m4!1s0x477eda5ba8617fd7:0x9805b678bb3387aa!8m2!3d45.4105663!4d11.8789403" >
```

Comment:
Forse in questo è il caso in cui l'immagine va messa perchè mi sembra abbastanza importante per lo scopo